dev/NavigationView/NavigationView.cpp

```diff
@@ -1399,6 +1400,32 @@ void NavigationView::OnItemsContainerSizeChanged(const winrt::IInspectable& send
     UpdatePaneLayout();
 }
 
+void NavigationView::UpdateOpenPaneWidth(double width)
```

Comment:
This probably needs to be called in OnApplyTemplate as well.

Reply:
SizeChanged is called on load, which in turn calls UpdateOpenPaneWidth. I tested it on load with a smaller window size and it updates openPaneLength properly.